pkg/cmd/admin/policy/modify_roles.go

```diff
@@ -78,6 +81,8 @@ func NewCmdAddRoleToUser(name, fullName string, f *clientcmd.Factory, out io.Wri
 	}
 
 	cmd.Flags().StringVar(&options.RoleNamespace, "role-namespace", "", "namespace where the role is located: empty means a role defined in cluster policy")
+	cmd.Flags().Var(&saNamesLong, "service-account", "service account in the current namespace to use as a user")
```

Comment:
> two-letter switch abbreviations are super-weird... --sa feels like -s -a

How many letters are required?  `sa` is used as an abbreviation for service accounts elsewhere, so choosing something else would be strange and I'm not sure I hate users enough to make them type the whole thing.